tests/integration/Core/Repository/URLWildcardService/CriterionTest.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Integration\Core\Repository\URLWildcardService;
+
+use eZ\Publish\API\Repository\Tests\BaseTest;
+use Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion;
+use Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\SearchResult;
+use Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\URLWildcardQuery;
+
+/**
+ * Test case criterion for URLWildcard.
+ *
+ * @see \eZ\Publish\API\Repository\URLWildcardService
+ * @group url-wildcard
+ */
+class CriterionTest extends BaseTest
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $urlWildcards = [
+            [
+                'source' => 'test',
+                'destination' => 'content-test',
+                'forward' => true,
+            ],
+            [
+                'source' => 'test test',
+                'destination' => 'content test',
+                'forward' => true,
+            ],
+            [
+                'source' => 'ibexa-dxp',
+                'destination' => 'ibexa-1-2-3',
+                'forward' => true,
+            ],
+            [
+                'source' => 'nice-url-seo',
+                'destination' => '1/2/3/4',
+                'forward' => false,
+            ],
+            [
+                'source' => 'no-forward test url',
+                'destination' => 'no/forward test url',
+                'forward' => false,
+            ],
+            [
+                'source' => 'Twitter',
+                'destination' => 'a/b/c',
+                'forward' => false,
+            ],
+            [
+                'source' => 'facebook',
+                'destination' => '2/3/facebook',
+                'forward' => true,
+            ],
+        ];
+
+        $repository = $this->getRepository();
+        $urlWildcardService = $repository->getURLWildcardService();
+
+        foreach ($urlWildcards as $urlWildcard) {
+            $urlWildcardService->create($urlWildcard['source'], $urlWildcard['destination'], $urlWildcard['forward']);
+        }
+    }
+
+    protected function doTestFindUrlWildcards(
```

Comment:
simply
```suggestion
    protected function findUrlWildcards(
```
would be enough